setup.php

```diff
@@ -82,7 +86,9 @@ function plugin_init_flyvemdm() {
 
    if ($plugin->isActivated('flyvemdm')) {
       require_once(__DIR__ . '/vendor/autoload.php');
-      require_once(__DIR__ . '/lib/GlpiLocalesExtension.php');
+      if (!class_exists('GlpiLocalesExtension')) {
+         require_once(__DIR__ . '/lib/GlpiLocalesExtension.php');
+      }
```

Comment:
this is needed to avoid conflict of the same class in several plugins. Can bo removed when GLPI will provide twig templating system